ember-cli/2017-10/october-19.md

```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-10-19
+
+## Attendees
+
+
+## Agenda
```

Comment:
agenda here

Reply:
ember-qunit update

Reply:
strategies RFCs

Reply:
ember-cli-update

Reply:
Where does Ember CLI use this library?

Reply:
console-ui

Reply:
console-ui has picked a non MIT license : https://github.com/aadsm/jschardet/blob/master/LICENSE in Ember 2.16. The bump was from greenkeeper. should we bump down and release 2.16.3?